Makefile

```diff
@@ -333,8 +333,8 @@ test-all: test-build test/gc/build/Release/binding.node
 test-all-valgrind: test-build
 	$(PYTHON) tools/test.py --mode=debug,release --valgrind
 
-CI_NATIVE_SUITES := addons addons-napi
-CI_JS_SUITES := abort async-hooks doctool inspector known_issues message parallel pseudo-tty sequential
+CI_NATIVE_SUITES ?= addons addons-napi
+CI_JS_SUITES ?= DEFAULT_JS_SUITES
```

Comment:
this is a magic variable which test.py understands, my thinking is that this would prevent us from accidentally forgetting to update this list in the future.

Reply:
If this removes `known_issues`, then we should explicitly add `known_issues` to `test-ci`. Alternatively, there can be a `JS_SUITES` variable and `CI_JS_SUITES` variable that is `JS_SUITES` + `known_issues`.

Reply:
@Trott I think this would be smart, main concern is I think we should remove `known_issues` from the coverage run, so split the configuration a bit between `test-ci` and `test-coverage`.